AGENTS.md

```
@@ -0,0 +1,149 @@
+## Contributing Notes (from CONTRIBUTING.md)
+- PRs against `development` branch
+- Include before/after screenshots for UI changes
+- No tests currently, but unit tests encouraged
+- Update docs (`USAGE.md`) with changes
+- Crash logs are obfuscated in release; deobfuscate with mapping.txt:
+  ```bash
+  $ANDROID_HOME/cmdline-tools/latest/bin/retrace app/build/outputs/mapping/tempusRelease/mapping.txt stack_error_transcript.txt
+  ```
+
+## Common Tasks
+| Task | Command |
```

markdownlint-cli2 (0.23.2)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)